src/public/app/widgets/note_detail.js

```diff
@@ -217,6 +217,9 @@ export default class NoteDetailWidget extends NoteContextAwareWidget {
 
     async beforeNoteSwitchEvent({noteContext}) {
         if (this.isNoteContext(noteContext.ntxId)) {
+            for (const x of this.children) {
```

Comment:
Is it intentional that the children array is never properly cleared after switching notes?

Reply:
Yes, it's there to keep e.g. CKEditor instance which is expensive to construct.